mrtt-ui/src/components/RestorationAimsForm/RestorationAimsForm.js

```diff
@@ -13,27 +13,30 @@ import { multiselectWithOtherValidation } from '../../validation/multiSelectWith
 import { questionMapping } from '../../data/questionMapping'
 import { restorationAims as questions } from '../../data/questions'
 import ButtonSubmit from '../ButtonSubmit'
-import CheckboxGroupWithLabelAndController from '../CheckboxGroupWithLabelAndController'
 import formatApiAnswersForForm from '../../library/formatApiAnswersForForm'
 import language from '../../language'
 import LoadingIndicator from '../LoadingIndicator'
+import RestorationAimsCheckboxGroupWithLabel from './RestorationAimsCheckboxGroupWithLabel'
 
+const getStakeholders = (data) =>
+  data.find((dataItem) => dataItem.question_id === '2.1')?.answer_value ?? []
 const RestorationAimsForm = () => {
   const { siteId } = useParams()
   const apiAnswersUrl = `${process.env.REACT_APP_API_URL}/sites/${siteId}/registration_answers`
   const [isLoading, setIsLoading] = useState(false)
   const [isSubmitError, setIsSubmitError] = useState(false)
   const [isSubmitting, setIsSubmitting] = useState(false)
+  const [stakeholders, setStakeholders] = useState([])
   const validationSchema = yup.object({
-    ecologicalAims: multiselectWithOtherValidation,
-    socioEconomicAims: multiselectWithOtherValidation,
+    // ecologicalAims: multiselectWithOtherValidation
+    // socioEconomicAims: multiselectWithOtherValidation,
     otherAims: multiselectWithOtherValidation
   })
   const reactHookFormInstance = useForm({
     defaultValues: {
-      ecologicalAims: { selectedValues: [], otherValue: undefined },
-      socioEconomicAims: { selectedValues: [], otherValue: undefined },
-      otherAims: { selectedValues: [], otherValue: undefined }
+      // ecologicalAims: { selectedValues: [], otherValue: undefined }
+      // socioEconomicAims: { selectedValues: [], otherValue: undefined },
+      otherAims: { selectedValues: [], otherValue: undefined, isOtherChecked: false }
```

Comment:
this didnt help, because when we load stuff from the api, isOtherChecked gets set back to undefined.